Svc/FpySequencer/FpySequencer.hpp

```diff
@@ -382,8 +380,10 @@ class FpySequencer : public FpySequencerComponentBase {
     // Run state
     // ----------------------------------------------------------------------
 
+    using Signal = FpySequencer_SequencerStateMachineStateMachineBase::Signal;
```

Comment:
Yeah, signal is defined within the `class` so in the C++ it would be `class::Signal`.  This is why it is not resuable